src/BlackSlope.Api.Common/BlackSlope.Api.Common.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">

```

Comment:
As part of this migration, we could start using  "Microsoft.NET.Sdk.Web" SDK instead and get rid of meta-package.

Reply:
Oh okay. I didn't because this was a supporting library and not the primary host executable, but I'll give it a try

Reply:
As I suspected all supporting libraries must use Microsoft.NET.Sdk and not Microsoft.NET.Sdk.Web, 
![image](https://user-images.githubusercontent.com/2224003/70933328-603eb580-2001-11ea-9545-28568ec0ff60.png)

I believe using Microsoft.NET.Sdk.Web enforces a Main method entry point check.

Reply:
I see that makes sense. Ignore my comment.